docs/source/introduction.rst

```diff
@@ -7,60 +8,108 @@ This section covers installation of pyChapel and required software packages,
 introduces the "pych" utility and provides a "hello world" example of using
 pyChapel.
 
+System Requirements
+~~~~~~~~~~~~~~~~~~~
+
+* Linux-64 host system (Ubuntu 16.04 is shown here). Mac OS-X is not yet supported.
+* Python-2.7 (the later, the better), with pip. Python-3 is not yet supported.
+* Support for Chapel "Quickstart" on the host. See `Prerequisites <https://chapel-lang.org/docs/latest/usingchapel/prereqs.html>`_
+
 Installation
 ~~~~~~~~~~~~
 
-The steps for installing pyChapel and the software packages it depends upon is
-fairly straightforward. The following provides a setup guide on a Ubuntu-linux system.
+The steps for installing pyChapel and the software packages it depends upon are
+fairly straightforward. The following provides a setup guide on a Ubuntu-16.04
+host system.
+
+The pyChapel Python module itself is available from PyPi.
+Make sure you are using the Python2 version of Pip and any other Python utilities.
+We recommend using Pip's "user install" option as shown here, but those familiar
+with other Python2 virtual environments may adapt these instructions accordingly.
+A Pip "user install" will install pyChapel under ``$HOME/.local``, so make sure that
+``$HOME/.local/bin`` is near the front of your PATH.
+
+Install pyChapel and dependencies from PyPi::
+
+  $ pip install --user "numpy>=1.14" "future>=0.15.2" pyChapel
+
+Run the pyChapel self-check. Note the "Missing Chapel lib" dir location::
+
+  $ pych -k     # if "command not found", make sure $HOME/.local/bin is in PATH
+    Checking installation...
+    * Templates
+    * Object Storage
+    * Libraries
+    ERR: Missing Chapel lib file(libchpl.a) in dir(/home/chapeluser/.local/share/pych/lib)
+    ERR: Missing Chapel lib file(main.o) in dir(/home/chapeluser/.local/share/pych/lib)
+    * Commands
+
+PyChapel requires the Chapel compiler and runtime, which must be built from source
+to work with pyChapel. 
+
+Get Chapel source from Github::
+
+  $ cd $HOME    # For example- any location that will not get accidentally deleted
+  $ git clone git@github.com:chapel-lang/chapel.git
+
+Note the location::
+
+    $HOME/chapel  (in this example)
 
-Depending on your system the commands might be slightly different but the 
-should provide the gist of what is required.
+Add the following to your shell environment::
 
-pyChapel depends on the pyChapel Python module itself which is available via
-PyPi. Additionally the Chapel compiler and a library compiled with the compiler
-is required.
+    (.bashrc, for example)
 
-Installing pyChapel::
+    #
+    # Setup Chapel environment
+    #
+    # CHPL_HOME=Chapel source location
+    #
+    export CHPL_HOME=$HOME/chapel   # in this example
+    export CHPL_LIBMODE=shared
+    source $CHPL_HOME/util/quickstart/setchplenv.bash
```

Comment:
Don't you still need to be in $CHPL_HOME to source the setchplenv scripts?  Or am I misremembering?

Reply:
Works as shown

Reply:
Sweeeeeeet